include/mbedtls/pk_info.h

```diff
@@ -0,0 +1,274 @@
+/**
+ * \file pk_info.h
+ *
+ * \brief Public Key cryptography abstraction layer: object interface
+ *
+ *  This file contains the info structure interface used by developers to
+ *  provide target-specific implementations of opaque key handling functions
+ *  (called engines in the following).
+ *
```

Comment:
Please cut the Doxygen part off here as has been done in 2.7 in other files.

Reply:
Done